shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -2272,11 +2289,57 @@ - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
   } else if ([method isEqualToString:kUpdateConfigMethod]) {
     [self updateConfig:args];
     result(nil);
+  } else if ([method isEqualToString:kOnPointerMoveMethod]) {
+    double pointerY = [args[@"pointerY"] doubleValue];
```

Comment:
use `CGRect` rather than `double` for UI stuff.

Reply:
With regards to the pointerY this only contains a single double, so I am not sure that this would classify as UI.

Reply:
sorry i meant CGFloat

Reply:
Gotcha, just updated the code!